src/NUnitFramework/tests/Constraints/CollectionEquivalentConstraintTests.cs

```diff
@@ -287,8 +287,10 @@ public void WorksWithNonIComparableTuples()
 
         // The following tests are each running in 14ms to 46ms on my machine. Based on that,
         // warn at 100ms and fail at 500ms
-        private const int LARGE_COLLECTION_WARN_TIME = 100;
-        private const int LARGE_COLLECTION_FAIL_TIME = 500;
+        // Seems to be slower on MacOs on build on github actions, so increasing this with 50%
+
+        private const int LARGE_COLLECTION_WARN_TIME = 100 * 3 / 2;
+        private const int LARGE_COLLECTION_FAIL_TIME = 500 * 3 / 2;
```

Comment:
If only MaCOS is the problem, don't run them there:
`[Platform(Exclude = "MACOSX", Reason = "MacOS github runner is very slow")]`

Reply:
Done